### DISCUSSION TITLE
1BCC44D2-D5C1-4427-8140-65D85BD39D74

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#120] [General]
# GQ Lupi

GQ Lupi is a binary system comprised of a young K-type T Tauri star and a red dwarf located about 495.17 lightyears from the solar system. The primary component of the system has a single super-Jupiter / brown dwarf at a projected distance of 100 au.
https://celestia.mobi/addon?item=1BCC44D2-D5C1-4427-8140-65D85BD39D74

**Comment by Karistus** [comment]
Hey, just a friendly reminder that copying Wikipedia's summaries and using them as descriptions in your addon is a violation of it's license (CC-BY-SA 4.0).

Some more advice as well, your descriptions of the planets you add should be short and to the point. No point in writing a short essay on your planet(s) unless your providing specific data about that planet(s) (i.e atmospheric composition).

**Reply by Galaxy-Leaper1** [reply]
I really shouldn't have plagiarized or copied，Due to my busy work schedule in the coming days，Almost no time.  Can you help me update the copy （Include NGTS-8）? I am very sorry for this behavior.

**Comment by kmjordan122313-lgtm** [comment]
gq lupi b is likely a brown dwarf, good addon tho.
i'd rate it 10/10
